dev/tools/localization/gen_l10n_templates.dart

```diff
@@ -65,10 +66,6 @@ import 'package:intl/intl.dart' as intl;
 /// be consistent with the languages listed in the @(class).supportedLocales
 /// property.
 abstract class @(class) {
-  @(class)(String locale) : assert(locale != null), _localeName = intl.Intl.canonicalizedLocale(locale.toString());
```

Comment:
This aspect of the generated code can't be removed, the formatters depend on it.

Reply:
Oops, I hadn't considered that. I'll add them back in. It looks like this is already failing the integration test :)